### ISSUE TITLE
Captcha

### ISSUE THREAD

**Issue by dereklieu** [developmentseed/sensor-rest-api#11]
@scisco are we ok going with [re-captcha](https://www.google.com/recaptcha/intro/index.html) for this?

**Comment by scisco** [comment]
@dereklieu yeah this should work. I just have to implement the verification side of it on the API.

However, I think this is lower priority now. I'd hold on until we have the rest of the API working.
